deps/v8/include/v8-version.h

```diff
@@ -17,4 +17,7 @@
 // (Boolean macro values are not supported by all preprocessors.)
 #define V8_IS_CANDIDATE_VERSION 0
 
+// Modify this when changes are made by the embedder.
+#define V8_EMBEDDER_VERSION "node.0"
```

Comment:
I think it's nicer to write this like this:
```c++
#ifndef V8_EMBEDDER_STRING
#define V8_EMBEDDER_STRING ""
#endif  // V8_EMBEDDER_STRING
```
And then override it at build time.  That would also make it more likely for this patch to get accepted upstream.